addons/s3/utils.py

```diff
@@ -143,3 +145,33 @@ def get_bucket_prefixes(access_key, secret_key, prefix, bucket_name):
             )
 
     return folders
+
+
+def update_folder_names(apps, schema_editor=None):
+    NodeSettings = apps.get_model('addons_s3', 'NodeSettings')
+
+    # Update folder_id for all records
+    NodeSettings.objects.update(folder_id=Concat(F('folder_id'), Value(':/')))
+
+    # Update folder_name for records containing '('
+    NodeSettings.objects.filter(folder_name__contains=' (').update(
+        folder_name=Replace(F('folder_name'), Value(' ('), Value(':/ ('))
+    )
+    NodeSettings.objects.exclude(folder_name__contains=' (').update(
+        folder_name=Concat(F('folder_name'), Value(':/'))
+    )
```

Comment:
Two questions:

* Do we need to update `folder_name`? I thought the name is used for display purpose only but I could be wrong.
* On staging server, does this migration break newly connected ones where `:/` is already in place?

Reply:
- Yes to maintain consistency because new names will have the colon
- Yes, hmm shouldn't be necessary but might want to fix for the staging server.